static/main.js

```diff
@@ -171,8 +171,10 @@ function replaceHTMLChar(char) {
 }
 
 function copyToClipboard(arg) {
-    var el = document.getElementById(arg).value
-    navigator.clipboard.writeText(el)
+    var el = document.getElementById(arg)
```

Comment:
I wrote this precisely because `document.execCommand` is deprecated.

Reply:
Mobile phone compatibility go brrrrr

Reply:
Oh :p. Well, let's leave it for now then :p